pyproject.toml

```diff
@@ -51,6 +51,7 @@ gladia = [ "websockets~=13.1" ]
 google = [ "google-generativeai~=0.8.3", "google-cloud-texttospeech~=2.17.2" ]
 gstreamer = [ "pygobject~=3.48.2" ]
 fireworks = [ "openai~=1.37.2" ]
+krisp = [ "pipecat_krisp~=0.1.0" ]
```

Comment:
Considering that we don't have this `pipecat_krisp` published, does it make sense to keep it here ?

Reply:
It doesn't. Package should probably be called `pipecat-ai-krisp`.